docs/developing/install/steps.rst

```diff
@@ -67,6 +72,16 @@ Run the database migrations:
 .. note::
     This will also create a sqlitedb file if it's missing
 
+Setting up the fixtures.
+There are two ways to do this.
+
+.. code:: shell
+
+    python manage.py load_fixtures
+
+If this works you can skip to create a superuser.
+If for some reason the command does not work you can try the old way to load the fixtures
```

Comment:
It isn't very clear what exactly are the two ways to load the fixtures are, and where the instructions of both methods start and end. You wouldn't want to load the fixtures twice!

Perhaps try to make it more clear with headings